vertx-template-engines/vertx-web-templ-handlebars/src/test/java/io/vertx/ext/web/templ/HandlebarsTemplateTest.java

```diff
@@ -305,4 +305,30 @@ public void testCachingEnabled(TestContext should) throws IOException {
     });
     test.await();
   }
+
+  @Test
+  public void testTemplatePerf(TestContext should) {
```

Comment:
It seems this was needed to verify things manually during development. Can you remove it?